core/src/main/resources/lib/form/radio.jelly

```diff
@@ -38,7 +38,7 @@ THE SOFTWARE.
     <st:attribute name="checked" />
     <st:attribute name="value" />
     <st:attribute name="id" />
-    <st:attribute name="onclick" />
+    <st:attribute name="onclick" deprecated="true">onclick handler. Deprecated; assign an ID and look up the element that way to attach event handlers.</st:attribute>
```

Comment:
may be better to use class name as repeatable elements don't work properly with IDs from what I remember.

IDs are safe to use in standalone pages / configuration forms that won't be used in repeatable elements